react-frontend/src/components/StyleComponents/nav.js

```diff
@@ -168,3 +177,79 @@ export const SkipToMainContent = styled.a.attrs({
     text-decoration: underline;
   }
 `;
+
+export const CenterBanner = styled.div.attrs({
+  className: 'centerBanner',
+})`
+  width: 1115px;
+  line-height: 25px;
+  @media screen and (max-width: 800px) {
+    .centerBanner {
+      display: flex;
+    }
+  }
+  @media screen and (min-width: 1205px) and (max-width: 1440px) {
+    margin-left: 1.5%;
+  }
+`;
+
+export const TopHeader = styled.div.attrs({
+  className: 'Banner',
+})`
+  position: fixed;
+  width: 100%;
+  z-index: 999;
+  display: flex;
+  background-color: #38598a;
+  justify-content: space-between;
+  height: 50px;
+  @media screen and (min-width: 1305px) and (max-width: 1440px) {
+    margin-left: -1%;
+  }
+`;
+
+export const CustomDrawer = styled(Drawer)`
+  .ant-drawer-body {
+    -ms-flex-positive: 1;
+    flex-grow: 1;
+    padding: 24px;
+    overflow: auto;
+    font-size: 14px;
+    line-height: 1.5715;
+    word-wrap: break-word;
+    background-color: #38598a;
+  }
+  height: 130px;
```

Comment:
Can we remove the fixed `height` here?

![ant-drawer-body](https://user-images.githubusercontent.com/16946297/97904263-146a2d00-1cf5-11eb-8e62-461b4dcd133d.png)

Reply:
If I remove the height:
![image](https://user-images.githubusercontent.com/20617793/97915572-872fd400-1d06-11eb-9e2f-63effdc35e1b.png)

Increasing the height to 140px to ensure similar top and bottom.